docs/pages/includes/config-reference/database-config.yaml

```diff
@@ -24,7 +24,6 @@ db_service:
     # 'redshift-serverless' - discovers and registers AWS Redshift Serverless databases.
     # 'elasticache' - discovers and registers AWS ElastiCache Redis databases.
     # 'memorydb' - discovers and registers AWS MemoryDB Redis databases.
-    # 'opensearch' - discovers and registers AWS OpenSearch Redis databases.
```

Comment:
`opensearch` does exist. https://github.com/gravitational/teleport/blob/73757d829da99c4f2d9773c221d7a5484d4ba539/api/types/matchers_aws.go#L61-L62

I guess we should add it in the next line. Thanks for catching this!